malariagen_data/plasmodium.py

```diff
@@ -189,6 +189,9 @@ def variant_calls(self, extended=False, inline_array=True, chunks="native"):
         # Add extended data
         if extended:
             data_vars = self._add_extended_data(root, inline_array, chunks, data_vars)
+            data_vars.pop(
+                "variant_altlen", None
+            )  # variant_altlen is under investigation in Pf8.zarr
```

Comment:
Perhaps it would be possible to handle this by removing the variable from the Pf8 config file, rather than here? In which case no changes to tests for pf7 or pv4 would be needed?

Reply:
Thanks, changed accordingly.